funsor/terms.py

```diff
@@ -443,7 +443,7 @@ def reduce(self, op, reduced_vars=None):
         assert reduced_vars.issubset(self.inputs)
         return Reduce(op, self, reduced_vars)
 
-    def sample(self, sampled_vars, sample_inputs=None):
+    def sample(self, sampled_vars, sample_inputs=None, rng_key=None):
```

Comment:
We should look at making `test_samplers.py` backend-agnostic as well to exercise this new code path, but that doesn't have to happen in this PR.

Reply:
Oh, thanks for pointing it out. I missed that test file.